docs/User-Guide.md

```diff
@@ -453,7 +456,64 @@ Outputs are additionally logged to /opt/intelmq/var/log/intelmqctl
 
 #### Botnet Concept
 
-The botnet represents all currently configured bots. To get an overview which bots are running, use `intelmqctl status`or use IntelMQ Manager.
+The botnet represents all currently configured bots which are explicitly enabled. To get an overview which bots are running, use `intelmqctl status` or use IntelMQ Manager. Set `"enabled": true` in the runtime configuration to add a bot to botnet. By default, bots will be configured as `"enabled": false` therefore, does not belong to the botnet due protection reasons. See [Bots](Bots.md) for more details on configuration.
+
+Disabled bots can still be started explicitly using `intelmqctl start <bot_id>`, but will keep marked as disabled if stopped. They are not started by `intelmqctl start` in analogy to the behavior of widely used initialization systems.
+
+
+#### Scheduled Run Mode
+
+In many cases it is useful to schedule a bot, for example, to collect information from a website every day at midnight. To do this, set `run_mode` to `scheduled` in the `runtime.conf` for the bot. Check the following example:
```

Comment:
"schedule a bot" -> "schedule a bot at a specific time (for example via cron(1)) ..."